app/views/patients/_form.html.erb

```diff
@@ -44,7 +44,6 @@
           <%= form.date_field :birth_date,
            class: 'input',
            placeholder: 'mm/dd/yyyy',
-           value: format_date(patient.birth_date),
```

Comment:
This needs to be fixed so that it works in Safari.

Reply:
Implemented a change...seems to resolve in both but should I focus on a different solution?